.github/workflows/publish-v2.yml

```diff
@@ -140,6 +140,8 @@ jobs:
                     ]
                   }
       - name: Remove git tag
-        uses : ./.github/workflows/remove-git-tag
+        uses: dev-drprasad/delete-tag-and-release@v0.2.0
```

Comment:
Use commit sha (and comment that it is v0.2.0)

Reply:
Done, point to :

https://github.com/dev-drprasad/delete-tag-and-release/commit/085c6969f18bad0de1b9f3fe6692a3cd01f64fe5